scraper/src/youtube2zim/scraper.py

```diff
@@ -84,6 +85,29 @@
 MAXIMUM_YOUTUBEID_LENGTH = 24
 
 
+class CustomIndexData(IndexData):
+    """Custom IndexData class to allow for custom title and content"""
+
+    def __init__(self, title: str, content: str):
+        self.title = title
+        self.content = content
+
+    def has_indexdata(self):
+        return True
+
+    def get_title(self):
+        return self.title
+
+    def get_content(self):
+        return self.content
+
+    def get_keywords(self):
+        return ""
+
+    def get_wordcount(self):
+        return 1
```

Comment:
You should compute the number of words, probably `return len(self.content.split()) if self.content else 0` would do the job

Reply:
Fixed in https://github.com/openzim/youtube/pull/265/files#diff-ba8e4270b357604b029038e7742f741b431b493a07d7482ea99e22b7002388cdR108